CRM/Activity/Form/Task/PDFLetterCommon.php

```diff
@@ -53,7 +53,7 @@ public static function createDocument($activityIds, $html_message, $formValues)
     $tp->addMessage('body_html', $html_message, 'text/html');
 
     foreach ($activityIds as $activityId) {
-      $tp->addRow()->context('activityId', $activityId);
+      $tp->addRow()->context('activity_id', $activityId);
     }
     $tp->evaluate();
 
```

Comment:
With a view to one day standardising more of this, how about:
```
    $entityField = CRM_Activity_Tokens::getEntityIDFieldName();
    foreach ($activityIds as $activityId) {
      $tp->addRow()->context($entityField, $activityId);
    }
```

Reply:
Hmm I commented below first so my comments may not make sense & also this comment supercedes the other..... If you look at my api PR I'm proposing removing this code inn favour of an api call

Reply:
The thought was that 'acitivity_id' or 'activityId' is defined once and avoids mismatch between these various uses.  But that's all void if we replace it anyway.